.ci_support/linux_python2.7.yaml

```diff
@@ -0,0 +1,9 @@
+pin_run_as_build: !!python/object/apply:collections.OrderedDict
+- - - python
+    - !!python/object/apply:collections.OrderedDict
```

Comment:
This is the problem.  Conda-smithy is outputting this decorated yaml.  It needs to add some yaml filtering so that these outputs are just plain yaml (no type annotation).